.travis.yml

```diff
@@ -27,6 +27,10 @@ matrix:
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.5
+    - os: osx
```

Comment:
Should also specify the macOS image to use. Ideally would be `xcode6.4` as that is one of the older ones available and has 10.9 SKD. So should create binaries that are widely deployable on macOS. FWIW conda-forge and IIUC MacPython do the same thing.

Reply:
ref: https://docs.travis-ci.com/user/reference/osx/#OS-X-Version

Reply:
thanks, updated!